test.md

```diff
@@ -0,0 +1 @@
+"This file should be ignored by git".
```

Comment:
- [ ] Kindly remove this file manually from your repo files, it's supposed to be ignored. This file is retained by GitHub even after you .gitignore it because it had already been pushed to Github before adding it to the .gitingnore file.

Reply:
## Deleted test.md